tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "paths": {
-      "@graphql-tools/utils": ["./node_modules/@graphql-tools/utils"]
+      "@graphql-tools/utils": ["./node_modules/@graphql-tools/utils"],
+      "graphql": ["./packages/supermassive/node_modules/graphql"]
```

Comment:
So we're using latest version for typings for all monorepo packages, but not for actual runtime?

Reply:
Yeah, that was the only way I managed to get it to run.